build-system/tasks/e2e/describes-e2e.js

```diff
@@ -31,6 +31,8 @@ const {
   SeleniumWebDriverController,
 } = require('./selenium-webdriver-controller');
 const {AmpDriver, AmpdocEnvironment} = require('./amp-driver');
+/* eslint-disable-next-line module-resolver/use-alias */
+const {configHelpers} = require('../../../testing/helpers');
```

Comment:
I assume we can't use `#testing` here because this directory isn't part of the runtime, and isn't babel-transformed, minified, etc.

Instead of disabling the lint rule like this, I think we should introduce an entry in `build-system/.eslintrc` that completely disables `module-resolver/use-alias` for all of `build-system/`.

/cc @rcebulko to keep me honest here.

Reply:
We actually already have one so I've gone ahead and updated it.

Reply:
Are we sure about that? We use `#alias` imports all over `test/`

Reply:
The files inside `test/` are babel transformed before they are run inside a browser using Karma + Mocha (unit / integration) or just Mocha (e2e). However, files in in `build-system/` are directly run in a nodejs environment without any babel transformation. I believe this is why alias imports work in test files but not in `build-system/`.

Reply:
Gotcha, thanks for the explanation